package.json

```diff
@@ -25,6 +25,7 @@
   "scripts": {
     "start-server": "./packages/server/index.js",
     "start-test": "./packages/test/index.js",
-    "test": "start-server-and-test start-server 4000/api/ start-test"
+    "test": "start-server-and-test start-server 4000/api/ start-test",
+    "install": "cd packages/cli/ && npm i && cd ../pages/ && npm i && cd ../server/ && npm i && cd ../test/ && npm i"
```

Comment:
Use `npm ci` instead of `npm i`, it will be faster.

Note: It should be used manually to install/update/uninstall dependencies because it relies on `package-lock.json`